scripts/generate_protobuf_and_funcs.rb

```diff
@@ -441,6 +441,39 @@ def generate!
   // Named tokens in scan.l
   #{@scan_protobuf_tokens.join("\n  ")}
 }
+
+message SummaryResult {
+  message SummaryResultTable {
+    string table = 1;
+    // FIXME(duckinator): I AM NOT CONFIDENT Node IS RIGHT. THIS NEEDS TO BE VERIFIED.
+    Node context = 2;
+  }
+  // tables: Vec<(String, Context)>
+  repeated SummaryResultTable tables = 1;
+
+  // aliases: HashMap<String, String>
+  map<string, string> aliases = 2;
+
+  // cte_names: Vec<String>
+  repeated string cte_names = 3;
+
+  // functions: Vec<(String, Context)>
+  message SummaryResultFunction {
+    string name = 1;
+    // FIXME(duckinator): Just like with SummaryResultTable, I am *NOT* confident using Node is right, and this needs to be verified.
+    Node context = 2;
+  }
+  SummaryResultFunction functions = 4;
+
+  // filter_columns: HashSet<(Option<String>, String)>
+  message SummaryResultFilterColumn {
+    // FIXME(duckinator): What should a and b be named?
+    // In the Rust version, they *aren't* named -- it's a Set of Tuples.
+    optional string a = 1;
+    string b = 2;
```

Comment:
In Rust, this is a `filter_columns: HashSet<(Option<String>, String)>`.

Notably, the inner parts are in a tuple so it's not named. I couldn't determine [from the context in pg_query.rs](https://github.com/pganalyze/pg_query.rs/blob/23ab4ae1c989fe0dea919d4fee043993e97f6223/src/parse_result.rs#L150-L161) what would make sense as a name.

Is `a` and `b` fine, or do you have better suggestions?

Reply:
They're table name and column name:
https://github.com/pganalyze/pg_query.rs/blob/23ab4ae1c989fe0dea919d4fee043993e97f6223/tests/filter_column_tests.rs#L10-L21

Which could either be `table` and `column`, or `table_name` and `column_name`